pages/GSoC-21.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: page
+title: Ideas page for GSoC 2021 | Hydraecosystem.org
+comments: true
+permalink: /gsoc-21
+---
+
+# Introduction
+Welcome to Hydra Ecosystem's ideas page for [GSoC 2021](https://summerofcode.withgoogle.com/). We would like to receive proposals that encompass different aspects of the development. Different tools are under development
+and candidates can present projects to fulfill multiple ideas on this page and issues listed at our Github [organization page](https://github.com/HTTP-APIs/).
+
+To get in touch with the Community:
+* [Slack](https://join.slack.com/t/hydraecosystem/shared_invite/enQtNzM3NTg5NzQ2MDUxLWU1MjM3ZGRhZWM4ZTg1ODBjMTljNTQwNzAwMGM3ZDlmYTY3Y2E4OGJmN2NlZWRjMWIzY2MzN2NjOTIyYmQ1ZjU)
```

Comment:
I think this link has expired. We need to generate a new one

Reply:
https://join.slack.com/t/hydraecosystem/shared_invite/zt-ly4j3hg3-gVhJxgc6ykpGrel_N4Xvvg